ct/calibre-server.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+source <(curl -s https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/misc/build.func)
+# Copyright (c) 2021-2024
+# Author: thisisjeron
+# License: MIT | https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+# Source: https://calibre-ebook.com
+
+# App Default Values
+APP="Calibre-Server"
+var_tags="eBook"
+var_cpu="2"
+var_ram="1024"
+var_disk="4"
+var_os="debian"
+var_version="12"
+var_unprivileged="1"
+
+# App Output & Base Settings
+header_info "$APP"
+base_settings
+
+# Core
+variables
+color
+catch_errors
+
+function update_script() {
+  header_info
+  check_container_storage
+  check_container_resources
+
+  # Check if installed
+  if [[ ! -f /etc/systemd/system/calibre-server.service ]]; then
+    msg_error "No ${APP} Installation Found!"
+    exit
+  fi
+
+  msg_info "Stopping ${APP}"
+  systemctl stop calibre-server
+  msg_ok "Stopped ${APP}"
+
+  msg_info "Updating ${APP} container packages"
+  apt-get update &>/dev/null
+  apt-get -y upgrade &>/dev/null
+  msg_ok "Container packages updated"
+
+  # Potentially re-run the official calibre script to ensure most recent version
+  # to keep logic consistent with how other scripts handle updates.
+  msg_info "Updating Calibre (latest)"
+  wget -nv -O- https://download.calibre-ebook.com/linux-installer.sh | sh /dev/stdin &>/dev/null
```

Comment:
If possible check if there is a newer version wich needs to be installed. Have a look at the snipeit.sh file for reference.